src/View/View.php

```diff
@@ -348,7 +348,7 @@ public function __construct(
         $this->request = $request;
         $this->response = $response ?: new Response();
         $this->Blocks = new $this->_viewBlockClass();
-        $this->initialize();
+        $this->initialize($viewOptions);
```

Comment:
I think this should be the diff of $viewOptions and keys set as view props similar to https://github.com/cakephp/cakephp/blob/d339441b1ce2a0164c41d3227f71f594558931c7/src/View/View.php#L337-L340.

Reply:
I think so too - if we want to make this change at all.

Reply:
config is available via `$this->getConfig()` so this Argument is useless in `View`

Reply:
The reason I made this is `$config` != $this->getConfig(). It is the original array from the constructor, not merged with defaults.

Reply:
if someone would need to `_passedVars`'s values , he can simply use like `$this->getLayout()` and other configs are available from `$this->getConfig()`
anyway I can not see good/reasonable usage for it

also there is not defaults in `View` , is there ?
infact , ```        'viewVars', 'autoLayout', 'helpers', 'template', 'layout', 'name', 'theme',
        'layoutPath', 'templatePath', 'plugin',```
are not at the output of `$this->getConfig().`

Reply:
The only reason is for consistency. I'm not a fan of $config, but we have it in other places.